authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,62 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
 
-"""Configure JWT Here"""
+from django.conf import settings
 
+from rest_framework import authentication, exceptions
+
+from .models import User, BlacklistedToken
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    """
+    This class gets the authorization header gotten
+    from the request header with the keyword Token in
+    it
+    """
```

Comment:
These changes are out of scope of your PT story. Make sure this PR only addresses the story assigned